src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,43 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void size() {
+       int result = numbers.size();
+       assertThat(result).isEqualTo(3);
```

Comment:
assertj에서 제공하는 컬렉션 검증 함수를 사용해 보셔도 좋을 것 같아요 😃
```suggestion
       assertThat(numbers).hasSize(3);
```